hw/formal/conn_csvs/top_earlgrey_conn.csv

```diff
@@ -0,0 +1,3 @@
+,NAME,SRC BLOCK,SRC SIGNAL,DEST BLOCK,DEST SIGNAL,,,,,,
+
+CONNECTION,CLKMGR_IDLE0,u_clkmgr_aon,idle_i[0],u_aes,idle_o
```

Comment:
Hi @matutem, this is just a trial from clock manager, feel free to expand or remove it.